### PR TITLE
Add icon for window

### LINKED ISSUE #59: Add icon on Linux
Currently on Linux there is no icon for the application.

### PR DESCRIPTION
**Pull request by keirlawson**
Fixes #59

### CODE REVIEW

File: src/browser/application.js [hunk 1]
```diff
@@ -80,9 +80,12 @@ class Application extends EventEmitter {
    * Opens up a new AppWindow and runs the application.
    */
   openWindow () {
+    let iconPath =  path.resolve(__dirname, '..', '..', 'static', 'img', 'yeoman-icon.png');
```
Comment (stefanbuck) [inline src/browser/application.js]: Did you tried to load the icon from `resources/app.png`?

**Comment by stefanbuck** [comment]
... Linux not Windows, right :wink: 

Please update the commit message. Thanks

**Comment by keirlawson** [comment]
Have changed in line with suggestions

**Comment by ruyadorno** [comment]
LGTM :+1:

**Pull request merged**